lib/gerry/client/groups.rb

```diff
@@ -13,10 +13,17 @@ def groups
 
       # Get all members for a group
       #
+      # @param [Array] options the query parameters
       # @return [Array] the members
-      def group_members(group_id)
+      def group_members(group_id, options = [])
```

Comment:
Are there other options besides recursive? I didn't see any mentioned in the doc.

If not, I'd prefer something like this:

```ruby
def group_members(group_id, recursive=false)
```